src/hooks/useAllCities.js

```diff
@@ -37,6 +41,10 @@ const useAllCities = ({ pageSize, sortOrder }) => {
     return { transformedData: items, total };
   }, [data]);
 
+  useEffect(() => {
```

Comment:
advice for the future, you can use a named function instead of an anonymous to improve the readability of the code

Reply:
Roger that!

Reply:
you can do this inline

```js
useEffect(
    function refetchCities() {
      refetch();
    },
    [searchQuery, refetch],
  );
```